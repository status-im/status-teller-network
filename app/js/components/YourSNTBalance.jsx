import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { Row, Col, Card } from 'reactstrap';
import { withNamespaces } from 'react-i18next';

class YourSNTBalance extends Component {
  render() {
    const t = this.props.t;
    return (
      <Row className="mt-5">
        <Col xs="12">
          <span className="font-weight-bold h5">{t('yourSNTBalance.label')}</span>
        </Col>
        <Col xs="12" className="pr-0">
          <Card body className="rounded-0">
            <span>{this.props.value} SNT</span>
          </Card>
        </Col>
      </Row>
    );
  }
}

YourSNTBalance.propTypes = {
  t: PropTypes.func,
  value: PropTypes.string
};

export default withNamespaces()(YourSNTBalance);
