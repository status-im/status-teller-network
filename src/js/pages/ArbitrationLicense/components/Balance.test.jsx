import React from "react";
import { shallow } from "enzyme";

import YourSNTBalance from "./Balance";

describe('YourSNTBalance', () => {
  it('should render correctly', () => {
    const component = shallow(<YourSNTBalance value={"10"}/>);
  
    expect(component).toMatchSnapshot();
  });
});
