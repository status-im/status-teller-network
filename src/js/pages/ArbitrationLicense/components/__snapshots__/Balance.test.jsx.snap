// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YourSNTBalance should render correctly 1`] = `
<Fragment>
  <h3 />
  <p
    className="border rounded p-3"
  >
    <span
      className="h3 font-weight-normal"
    >
      <img
        alt="SNT icon"
        className="mr-2"
        src="test-file-stub"
      />
      10
       SNT
    </span>
  </p>
</Fragment>
`;
